src/content/en/2020/performance.md

```diff
@@ -340,39 +360,41 @@ Likening this years’ TTFB geo readings to [2019 results](https://almanac.httpa
+There are dozens of different user-centric metrics that can be used to assess websites and applications. However, sometimes the predefined metrics don't quite fit our specific scenarios and needs. The [PerformanceObserver API](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver) allows us to obtain custom metric data obtained with [User Timing API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API), [Long Task API](https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API), [Event Timing API](Event Timing API) and [a handful of other low-level APIs](https://web.dev/custom-metrics/). For example, with their help, we could record the timing transitions between pages or quantify server-side-rendered (SSR) application hydration.
 
 {{ figure_markup(
   image="performance-performance-observer-usage.png",
   caption="Performance Observer usage by device type.",
   description="Bar chart showing that the adoption of PErformance Observer API is low, at 6.6% on desktop and 7% on mobile.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=632678090&format=interactive",
-  sheets_gid="2077755325",
+  sheets_gid="934401790",
   sql_file="performance_observer.sql"
   )
 }}
 
-The chart above showcases that Performance Observer, despite being available for a while now (and [widely supported cross-browser](https://caniuse.com/mdn-api_performanceobserver)) is **only used by 6-7% of tracked sites**, depending on device type. Those websites will be leveraging the low-level APIs to create custom metrics, and the PerformanceObserver API to collate them, and then potentially use it with other performance reporting tooling. Such low adoption rates might indicate the tendency to lean on predefined metrics (for example, coming from Lighthouse), but also the yet to be leveraged potential of the API.
+The chart above showcases that Performance Observer, despite being available for a while now (and [widely supported cross-browser](https://caniuse.com/mdn-api_performanceobserver)) is only used by 6-7% of tracked sites, depending on device type. Those websites will be leveraging the low-level APIs to create custom metrics, and the PerformanceObserver API to collate them, and then potentially use it with other performance reporting tooling. Such low adoption rates might indicate the tendency to lean on predefined metrics (for example, coming from Lighthouse), but also the yet to be leveraged potential of the API.
+
+{# TODO - that's actually higher than I expected as it's a bit niched. Shjould we frame this in a more positivie light as per out Glass Hlaf Full principal (https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#principles)? Also any thoughts on why this is so high? #}
```

Comment:
@thefoxis thoughts?

Reply:
@bazzadp sure. we can rephrase to:

`The chart above showcases that Performance Observer is used by 6-7% of tracked sites, depending on device type. Those websites will be leveraging the low-level APIs to create custom metrics, and the PerformanceObserver API to collate them, and then potentially use it with other performance reporting tooling. Such adoption rates might indicate the tendency to lean on predefined metrics (for example, coming from Lighthouse), but also are impressive for a relatively niche API.`

Reply:
I like it, will change.

One more question: are you aware of any popular RUM tools or any platforms that have these built in which might explain why 6-7% of sites are using this? It seems too high to be manually coded.

Reply:
hmm, I'm not aware of any. seems like [speedcurve has them, to some degree](https://speedcurve.com/blog/user-timing-and-custom-metrics/), but unsure if that's the exact way you're picturing it :) we have people asking about custom metrics, but very occasionally hence it's not available in calibre (yet)

Reply:
Yeah that's the sort of thing I was thinking of. Or CDN tools like Boomerang or mPulse which might explain this - so if you're an Akamai customer do you get those automatically? We know from last year that [Akamai are 5% of third-party](https://almanac.httparchive.org/en/2019/cdn#fig-7) so that might go some way to explaining this number.

Anyway, probably delving into the weeds too much here for a Copyedit but was curious because, as I said, that was higher than I expected for being manually added.

Will change to above wording and close this. Thanks for discussing!

Reply:
Changed to suggested text.